vanilla-framework/vanilla-framework
# Fix form validation docs to display correct classes

## Done

- Point docs at correct example partial
- Update docs to specify correct state classes
- Remove redundant 'feedback' example partial

## QA

- Check code
- Run docs locally - `documentation-builder --base-directory docs --output-path docs/build`
- Ensure `/en/base/forms/validation-classes` displays correctly

## Details

Fixes #1003
